tests/test_base.py

```diff
@@ -278,7 +292,7 @@ def test_get_token_file(self):
 
     def test_get_token_file_empty(self):
         """ Test getting a token from a file with just whitespace. """
-        token_in_file = "   "
+        token_in_file = "   "  # nosec
```

Comment:
I think this slipped in from your other PR https://github.com/psss/did/pull/297 .
```suggestion
        token_in_file = "   "
```